mapper/Cargo.toml

```diff
@@ -0,0 +1,20 @@
+[package]
+name = "mapper_messages"
```

Comment:
Just call this mapper? Unless there's a namespace conflict?

Reply:
Yes, I think there will be a namespace conflict when the application on spot-firmware called mapper reaches for it.